src/pages/Login/components/Login/LoginComponent.tsx

```diff
@@ -41,41 +45,49 @@ const LoginComponent = () => {
         navigate("/", { replace: true })
       })
       .catch(() => {
-        alert("잘못된 이메일이거나 잘못된 비밀번호의 조합입니다.")
+        setAlertMessage("잘못된 이메일이거나 잘못된 비밀번호의 조합입니다.")
+        showModal()
       })
   }
 
   return (
-    <S.LoginComponentLayout>
-      <S.LoginComponentTitle>로그인</S.LoginComponentTitle>
-      <S.LoginForm onSubmit={handleLogin}>
-        <LoginInputContainer
-          updateUserInfo={updateUserInfo}
-          type="email"
-          placeholder="이메일"
-        />
-        <LoginInputContainer
-          updateUserInfo={updateUserInfo}
-          type="password"
-          placeholder="비밀번호"
-        />
-        <S.ButtonContainer>
-          <S.Button
-            type="submit"
-            $color={theme.colors.point}
-          >
-            로그인
-          </S.Button>
-          <S.Button
-            type="button"
-            $color={theme.colors.sub_alt}
-            onClick={() => navigate("/signup")}
-          >
-            회원가입
-          </S.Button>
-        </S.ButtonContainer>
-      </S.LoginForm>
-    </S.LoginComponentLayout>
+    <>
+      <ModalAlert
+        isShow={isShowModal}
+        alertMessage={alertMessage}
+        onClose={closeModal}
+      />
```

Comment:
[P2] 요렇게 감싸주시면 렌더링시 해당 값이 false일때는 매번 렌더링 하는 부분이 없어지지 않을까 생각해봅니다
```suggestion
{ isShowModal &&
      <ModalAlert
        isShow={isShowModal}
        alertMessage={alertMessage}
        onClose={closeModal}
      />
      }
```

Reply:
좋은 의견 감사합니다!! 😊